### ISSUE TITLE
Add information about how to get a bot listed on the site

### ISSUE THREAD

**Issue by MattyIce** [MattyIce/bottracker#25]

**Comment by chhantyal** [comment]
Looks like this has to be tackled as well. I see there is one issue open #28 

I also tried to add my bot https://steemit.com/@trendingbot, no luck yet. May be it's rejected because it's too small?

**Comment by MattyIce** [comment]
Hey @chhantyal - I manually review bots before adding them to the site. I will review yours shortly. I am working on some major changes to the site right now and will try to get this in along with those.

Also your bot will need to have at least 1,000 SP in the account (or delegated to the account) to include it in the bid-bots section.

**Comment by chhantyal** [comment]
I see, that is fair enough I guess.

However, I see many other bots on `Other bots` section which have less SP. That's where I am looking for now :D

**Comment by goktug0814** [comment]
Hello. How do I add my @sebil account to the steembottracker site? I have 1032 sp in my account. Thanks.

**Comment by sebil** [comment]
Hey @goktug0814, I think you mistakely mentioned me, am I right?

**Comment by goktug0814** [comment]
no, I have a steemit account called sebil.  I want to trade on www.steembottracker.com.  Looking for friends to help. Mattylce, Thanks

**Comment by Transisto** [comment]
Maybe list your api url and relevant detail at 1000 SP you won't show up in
the top bidbot category.
